### PR TITLE
Prepare PyPi distribution for release 0.3.0

### PR DESCRIPTION
**Pull request by ckadner**
**Description of your changes:**

Documentation update for the 0.3.0 release on PyPi.

Preview release: https://test.pypi.org/project/kfp-tekton/0.3.0rc5/

**Environment tested:**

* Python Version (use `python --version`): 3.7.5
* Tekton Version (use `tkn version`): 0.15
* Kubernetes Version (use `kubectl version`): 1.16
* OS (e.g. from `/etc/os-release`): Mac OS 10.15

/cc @animeshsingh

### CODE REVIEW

**Comment by ckadner** [comment]
@Tomcli -- I removed the "Optional Features" section and moved the "Building Tekton from Master" paragraph to the `FEATURES.md` doc just as an informational section since we don't currently rely on Tekton master for any of our features

**Comment by Tomcli** [comment]
> @Tomcli -- I removed the "Optional Features" section and moved the "Building Tekton from Master" paragraph to the `FEATURES.md` doc just as an informational section since we don't currently rely on Tekton master for any of our features

Can we not put it under FEATURES? For 0.3 release we don't rely anything on master, so it could confuse people who are new to this repo. Can we put it under developer guide instead?

**Comment by ckadner** [comment]
> > @Tomcli -- I removed the "Optional Features" section and moved the "Building Tekton from Master" paragraph to the `FEATURES.md` doc just as an informational section since we don't currently rely on Tekton master for any of our features
> 
> Can we not put it under FEATURES? For 0.3 release we don't rely anything on master, so it could confuse people who are new to this repo. Can we put it under developer guide instead?

Sure. Done.

**Comment by ckadner** [comment]
@animeshsingh -- If you are okay with the changes to the SDK README and how it shows up on Test-PyPI (https://test.pypi.org/project/kfp-tekton/0.3.0rc5/#description) then I will upload the `0.3.0` release to the real Python Package Index

File: sdk/README.md [hunk 1]
```diff
@@ -133,86 +130,21 @@ in the same directory:
 
 After compiling the `sdk/python/tests/compiler/testdata/parallel_join.py` DSL script
 in the step above, we need to deploy the generated Tekton YAML to our Kubernetes
-cluster with `kubectl`. The Tekton server will automatically start a pipeline run
-for which we can follow the logs using the `tkn` CLI. 
+cluster with `kubectl`. The Tekton server will automatically start a pipeline run.
+We can follow the logs using the `tkn` CLI.
```
Comment (animeshsingh) [inline sdk/README.md:134]:
```suggestion
We can then follow the logs using the `tkn` CLI.
```

File: sdk/README.md [hunk 2]
```diff
@@ -133,86 +130,21 @@ in the same directory:
...
-[master](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#install-pipeline)
-branch. Features that require a special build, different from the 'Tested Version',
-will be listed below.
-
-- [Exit Handler](/sdk/FEATURES.md#exit-handler)
-
-
-## Optional Features
-
-### Compile Kubeflow Pipelines without Artifacts
```
Comment (animeshsingh) [inline sdk/README.md:185]: We are removing this because this moved to server side 
cc @Tomcli

Comment (Tomcli) [reply sdk/README.md:185]: yes, I discussed with Christian and this is no longer needed (for SDK).

Comment (ckadner) [reply sdk/README.md:185]: we originally had this section for:
- generate optional PipelineRun ... no longer optional
- enable (later disable) artifacts ... server side now

**Comment by ckadner** [comment]
Thanks @Tomcli @animeshsingh 

The v`0.3.0` on **PyPI**: https://pypi.org/project/kfp-tekton/0.3.0/

One-line install: `pip install kfp-tekton`

**Pull request merged**